prachi_workspace/assignment/library_management_system.py

```diff
@@ -0,0 +1,85 @@
+
+##/////////////////////.......  LIBRARY MANAGEMENT SYSYTEM  ........////////////////////
+
+def add_data(title,author,book_id,availability=True):
+   file = open("text.txt","a")
+   file.write(f"{title},{author},{book_id},{availability}\n")
+   file.close()
```

Comment:
Read the concept of how to do file handling with `with open`  and try to implement it 

Ex:- 
with open('example.txt', 'a') as file:
    ..................................
    ................................